IPNI.md

```diff
@@ -463,6 +463,11 @@ and uses the multihash portion of the CID only.
 * `cid` - _Required_. The default string representation of the Cid. Currently, Base32 is used for
   CIDv1 as the encoding for the multibase string and Base58 is used for CIDv0.
 
+##### Query Parameters
+
+* `cascade` - _Optional_. The comma separated alternative routing systems to which the lookup is cascaded in addition to searching IPNI index records. If unspecified, only IPNI index records are searched. Supported values:
+    * `ipfs-dht`- The IPFS Kademlia DHT.
```

Comment:
I assume you mean DHT created by self-[identifying](https://github.com/libp2p/specs/tree/master/identify#protocols) peers that speak `/ipfs/kad/1.0.0` libp2p protocol?

Mind adding clarification, which DHT you mean? (Kubo has at least two, we may have more in the future when if double hashing is exposed as separate DHT).

Reply:
> I assume you mean DHT created by self-[identifying](https://github.com/libp2p/specs/tree/master/identify#protocols) peers that speak /ipfs/kad/1.0.0 libp2p protocol?

Good question. The answer is: Sort of. Allow me to explain:

The idea here is to provide _labels with implicit meaning_ instead of concrete protocol IDs.
The mapping between the two _could be 1:1_; but not necessarily. It offers a user-friendly flex point in the protocol where we can use as a tuning knob, adding labels and deprecating exiting ones to fit what's contextually needed by the users.

In this case what is meant by the label `ipfs-dht` could be one or more protocol IDs. For example, if/when the double hashing DHT materialises the label would span to automatically include the new DHT protocol ID and "just work" by inferring which specific DHT it should do lookups in for a given multihash by checking the multihash code. 

What are your thoughts? Does that make sense?
Would it be clearer if it's changed to `ipfs` or `kubo`?

Reply:
No, there is nothing specific to  `kubo` in `/ipfs/kad/1.0.0`.

> The idea here is to provide labels with implicit meaning instead of concrete protocol IDs. 

Ack, then `ipfs-dht` is fine :+1:  Thanks for explanation!